intranet/static/themes/afd2016/memes.js

```diff
@@ -2,15 +2,18 @@ $(function() {
     memeBinds = function() {
         $(".meme-options .meme-option").click(function() {
             var was_selected = $(this).hasClass("selected");
-            var this_id = $(this).attr("data-meme");
             $(".meme-options .meme-option").removeClass("selected");
-            if(was_selected) {
-                $(".meme-submit").prop("disabled", "disabled");
-                return;    
+
+            if (was_selected) {
+                $(".meme-submit").prop("disabled", true);
+                return;
             }
+
+            var this_id = $(this).attr("data-meme");
+
             $(this).addClass("selected");
-            $(".meme-hidden").attr("value", this_id);
-            $(".meme-submit").prop("disabled", "");
+            $(".meme-hidden").val(this_id);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2747525225/issues/source?bid=3212528&fileBranchId=3355681#l15)